### PR TITLE
Added login data and used k8s js client to get data

### PR DESCRIPTION
**Pull request by vithati**

### CODE REVIEW

File: armTemplates/windows-webapp-template.json [hunk 1]
```diff
@@ -0,0 +1,187 @@
+{
```
Comment (madhuv-msft) [inline armTemplates/windows-webapp-template.json]: please remove extra blank lines, you can use the following for the same
https://raw.githubusercontent.com/Microsoft/devops-project-samples/master/node/express/webapp/armTemplates/windows-webapp-template.json

File: src/pages/login/login.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+<html>        
+<head>
+    <style>
+    .login_form {
+            position: absolute;
+            top: 20%;
+            left: 30%;
+            right: 30%;
+            bottom: 20%;
+            font-size: 18px;
```
Comment (madhuv-msft) [inline src/pages/login/login.html]: low prio: ideally we should plan to use our common css for font rather hard coding the size,

File: src/server.js [hunk 3]
```diff
@@ -1,124 +1,145 @@
...
     }
+
+    return false;
 }
 
 function getUserInput(req) {
     var urlObject = url.parse(req.url, true);
     var requestUrl = urlObject.pathname.toLowerCase();
     var namespaceComputed = urlObject.query["namespace"] || userInputNamespace || defaultNamespace;
+    console.log('getUserInput requestUrl:' + req.url + ' namespace: ' + namespaceComputed);
```
Comment (madhuv-msft) [inline src/server.js]: for req.url use requestUrl

File: src/server.js [hunk 4]
```diff
@@ -1,124 +1,145 @@
...
+    processCommands(req, res, getNamespaces);
+});
+
+app.get('/getdeployments', function(req, res) {
+    processCommands(req, res, getDeployments);
+});
+
+app.get('/getreplicasets', function(req, res) {
+    processCommands(req, res, getReplicaSets);
+});
```
Comment (madhuv-msft) [inline src/server.js]: low prio: all of these commands can be a dict,

**Comment by vithati** [comment]
No blocking issues. I will address comments in another PR.

**Pull request merged**